src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs

```diff
@@ -92,7 +92,7 @@ void Initialize()
 			TranslatesAutoresizingMaskIntoConstraints = false
 		};
 
-		foreach (var view in Children)
+		foreach (var view in children)
```

Comment:
Will this cause issues if the collection changes during this loop? Before it was a copy, now it is the actual collection and IEnumerable throws if you change it while it is iterating.

Reply:
It's a possibility, but the likelihood is slim. 

As Vlad pointed out in our standup today that Snackbar + Toast typically only contain 2-3 children, so enumerating through the list should be very quick. The odds of the `List` being modified during enumeration are very low.

Reply:
Prior to this change, the foreach loop accessed the Children property, which calls children.ToFrozenSet(). ToFrozenSet() has to iterate over the collection.

In either case, AlertView is a UI control and should only ever be modified on the UI thread. If another thread is adding children to the view, that's a bug in the consumer of AlertView.